lib/path.js

```diff
@@ -285,9 +285,7 @@ const stringifyArgs = (command, args, precision, disableSpaceAfterFlags) => {
       result += roundedStr;
     } else if (
       !Number.isInteger(previous) &&
-      rounded != 0 &&
-      rounded < 1 &&
-      rounded > -1
+      !'0123456789'.includes(roundedStr[0])
```

Comment:
This file had a `isDigit` function defined. Could you use that instead?

```suggestion
      !isDigit(roundedStr[0])
```

This should be the same thing, but runs notably faster.

<details>

<summary> Benchmark: </summary>

```js
const Benchmark = require('benchmark');
const suite = new Benchmark.Suite;

const data = [
  '0',
  '1',
  '-',
  '.',
]

const isDigit = (c) => {
  const codePoint = c.codePointAt(0);
  if (codePoint == null) {
    return false;
  }
  return 48 <= codePoint && codePoint <= 57;
};

suite.add('#includes', () => {
  data.map((d) => '0123456789'.includes(d));
})
.add('between chars', () => {
  data.map((d) => d >= '0' && d <= '9');
})
.add('between chars v2', () => {
  data.map(isDigit);
})
.on('cycle', (event) => {
  console.log(String(event.target));
})
.on('complete', function() {
  console.log('Fastest is ' + this.filter('fastest').map('name'));
})
.run({ 'async': true });
```

```
#includes x 27,552,035 ops/sec ±1.06% (92 runs sampled)
between chars x 40,564,628 ops/sec ±1.23% (94 runs sampled)
between chars v2 x 57,363,760 ops/sec ±0.55% (94 runs sampled)
Fastest is between chars v2
```

</details>